hooks/__hooks__.py

```diff
@@ -1,18 +1,7 @@
-''' __hooks__.py
+from flat_schedule import create_flat_schedule
+from guidebook import create_csv_schedule
+from schedule_summary import create_summary_schedule
 
-Attach Python functions to wok hooks.
-'''
-
-import flat_schedule
-import guidebook
-import schedule_summary
-
-# The `hooks` dictionary that wok will import
-# See http://wok.mythmon.com/docs/hooks/
 hooks = {
-    'site.done': [
-            flat_schedule.create_flat_schedule,
-            guidebook.create_csv_schedule,
-            schedule_summary.create_summary_schedule,
-        ],
-    }
+    'site.done': [create_flat_schedule, create_csv_schedule, create_summary_schedule],
+}
```

Comment:
I have no idea what the hell went on here. I must have deleted the file by mistake somehow because it wasn't on my repo, I'll reset it.